java/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+   - name: postgresql
+     version: ~2.6.5
```

Comment:
this is an old version..

Reply:
I wanted to be in Sync with repo that uses Postgres.
https://github.com/hmcts/bulk-scan-processor/blob/master/charts/bulk-scan-processor/requirements.yaml

I have now updated the version. Thanks